process/matchers.go

```diff
@@ -140,20 +137,20 @@ func IsRunningCmdWithGID(cmd string, groupname string) error {
 
 // ListRunning returns a slice of all running processes
 // It returns an error of a process status could not be obtained
-func ListRunning() ([]*resource.Process, error) {
-	var ps []*resource.Process
-	err := withRunningProcs(func(procs procfs.Procs) error {
-		for _, proc := range procs {
-			pstat, err := proc.NewStat()
-			if err != nil {
-				return err
-			}
-			ps = append(ps, &resource.Process{
-				Pid: proc.PID,
-				Cmd: pstat.Comm,
-			})
-		}
-		return nil
-	})
-	return ps, err
-}
+//func ListRunning() ([]*Process, error) {
```

Comment:
```process``` package rework will be in a separate branch. This was just transferred over form the original code.